[RHCLOUD-35780] Make Drawer notifications actionable by adding a link to additional details (#4405)
RedHatInsights/notifications-backend: common-template/src/main/resources/templates/drawer/Sources/availabilityStatusBody.md

> [{data.context.resource_display_name}]({environment.url}/settings/sources/detail/{data.context.source_id}?from=notifications&integration=drawer)'s availability status was changed from **{data.context.previous_availability_status}** to **{data.context.current_availability_status}**.

Reviewer: ```suggestion
The availability of integration [{data.context.resource_display_name}]({environment.url}/settings/sources/detail/{data.context.source_id}?from=notifications&integration=drawer) changed from **{data.context.previous_availability_status}** to **{data.context.current_availability_status}**.
```
